certificate-policy.md

```diff
@@ -600,90 +599,117 @@ The CA SHALL disclose all Cross Certificates that identify the CA as the Subject
 ## 4.1 Certificate Application
 
 ### 4.1.1 Who can submit a certificate application
-In accordance with Section 5.5.2, the CA SHALL maintain an internal database of all previously revoked Certificates and previously rejected certificate requests due to suspected phishing or other fraudulent usage or concerns. The CA SHALL use this information to identify subsequent suspicious certificate requests.
+In accordance with Section 5.5.2, all CAs SHALL maintain an internal database of all previously revoked Certificates and previously rejected certificate requests due to suspected phishing or other fraudulent usage or concerns. All CA SHALL use this information to identify subsequent suspicious certificate requests.
+
+For the Root and Subordinate CAs:
+- An application for a CA certificate shall be submitted by an authorized representative of the applicant CA.
+
+For end entity certificates:
+- A certificate application shall be submitted to the CA by the Subscriber, an authorized organization representative, or an RA on behalf of the Subscriber. 
+
 
 ### 4.1.2 Enrollment process and responsibilities
-Prior to the issuance of a Certificate, the CA SHALL obtain the following documentation from the Applicant:
+For the Root and Subordinate CAs:
+- The Policy Authority is responsible for approving or denying requests for CA certificate issuances.
+
+For all CAs, prior to the issuance of any Certificate, the CA SHALL obtain the following documentation from the Applicant:
 
 1. A certificate request, which may be electronic; and
 2. An executed Subscriber Agreement or Terms of Use, which may be electronic.
 
-The CA SHOULD obtain any additional documentation the CA determines necessary to meet these Requirements.
+The certificate request SHALL contain a request from, or on behalf of, the Applicant for the issuance of a Certificate, and a certification by, or on behalf of, the Applicant that all of the information contained therein is correct.
+
+The CA SHALL be responsible for validating the information in the certificate request and the identity evidence to ensure the information is: 
 
-Prior to the issuance of a Certificate, the CA SHALL obtain from the Applicant a certificate request in a form prescribed by the CA and that complies with this CP. One certificate request MAY suffice for multiple Certificates to be issued to the same Applicant, subject to the aging and updating requirement in Section 3.3.1, provided that each Certificate is supported by a valid, current certificate request signed by the appropriate Applicant Representative on behalf of the Applicant. The certificate request MAY be made, submitted and/or signed electronically.
+- properly formed
+- accurate
+- meets the requirements for the type of certificate requested: a device Domain Validation SSL end entity certificate, a device Organizational Validation SSL end entity certificate, a CA Certificate, or a Certificate Status Server (OCSP) signing certificate
```

Comment:
A couple methods that jump out in 3.2.2.3 where we could auto-OV

> A third party database that is periodically updated and considered a Reliable Data Source;

So in theory we could use the .gov domain list and its `Type` field to do this to distinguish between federal and non-federal .gov services.

As stated earlier I have serious concerns with hinging our audits on that list's quality with how it's managed at present. However, if it was the difference between OV certs being fully automated with no additional work on top of DV and not, then it would be worth maturing that process on DotGov's side.